WeatherApp/Globals/UIButton+Extensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIButton+Extensions.swift
+//  WeatherApp
+//
+//  Created by Lucija Balja on 25/08/2020.
+//  Copyright © 2020 Lucija Balja. All rights reserved.
+//
+
+import UIKit
+
+extension UIButton {
+    
+    func style(size: CGFloat = 25.0, isBold: Bool = true, title: String) {
```

Comment:
I also think that it might make more sense to create a UIButton subclass (something like PrimaryButton), which would then be initialized with these attributes. 
The reason is that this func defines a specific button appearance, which is not apparent from the func name nor from the class it's called on. 
Another option could be to rename the func to something like "applyDefaultStyle(...)" . That's also a good alternative in my opinion.